src/Commands/SitemapCommand.php

```diff
@@ -64,7 +71,7 @@ protected function crawlWebsite($url)
 
         // Add a URI discoverer. Without it, the spider does nothing.
         // In this case, we want <a> tags and the canonical link
-        $spider->getDiscovererSet()->set(new XPathExpressionDiscoverer("//a|//link[@rel=\"canonical\"]"));
+        $spider->getDiscovererSet()->set(new XPathExpressionDiscoverer("//div[@rel=\"canonical\"]//a"));
```

Comment:
Hey @DmitrySidorenkoShim,

what is the plan here? The selector wasn't working for you?

Reply:
@DmitrySidorenkoShim